webui/settings.py

```diff
@@ -25,7 +25,7 @@
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
```

Comment:
What does this comment mean? cc: @asmorodskyi

Reply:
I bet is default comment generated by Django